metaflow/plugins/argo/argo_workflows.py

```diff
@@ -227,8 +227,8 @@ def delete(name):
 
         return schedule_deleted, sensor_deleted, workflow_deleted
 
-    @staticmethod
-    def terminate(flow_name, name):
+    @classmethod
```

Comment:
does this need to be a class method if the `cls` is not accessed?

Reply:
I was making it compatible with what I had for the `terminate` method in `step_functions.py`